Cargo.toml

```diff
@@ -23,8 +23,10 @@ serde_derive = "1"
 serde_json = "1.0"
 rust-crypto = "0.2" 
 sha2 = "0.8.0"
+sha3 = "0.8.2"
 dirs = "1.0.4"
 regex = "1"
+linefeed = "0.6"
```

Comment:
This package scares me.  It has a lot of `unsafe` behavior, and its main dependency, `mortal`, has a _ton_ of `unsafe` behavior.  Is there any other way to deal with this?